### PR TITLE
create CustomerHome and ShopOwnerHome

### PR DESCRIPTION
**Pull request by KarishmaSinghChauhan**

### CODE REVIEW

File: frontendservice/src/views/CustomerHome.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div class="home">
+    <h1>Welcome customer!</h1>
+    <p>
+      This page will be Home page for the customers. There will be a navbar
+      having the required navigation links myTokens etc and allShops will be
+      displayed with Search feature. Bavanya will be implementing this part.
```
Comment (priytanvani) [inline frontendservice/src/views/CustomerHome.vue:15]: Let's try not to check-in names of expected contributors here.
If you want, you can add TODOs as comments where you can mention feature/bugs pending.

File: frontendservice/src/views/ShopOwnerHome.vue [hunk 2]
```diff
@@ -8,40 +8,39 @@ specific language governing permissions and limitations under the License. */
...
-    <h1>Welcome!</h1>
-    <p v-if="loggedIn">Your username is {{ userName }}</p>
-    <br>
-    <p>This page will be common for all our users - customers as well as shop-owners. There will be a navbar having the required navigation links myShops, createShop, myTokens links depending on whether the user is loggedIn or verified.
-    <br>
+    <h1 v-if="isLoggedIn">Welcome! {{ name }}</h1>
+    <p>
+      This page will be Home page for the shop-owners. There will be a navbar
+      having the required navigation links myShops, createShop etc depending on
+      whether the user is verified or not. Ishita will be implementing this
```
Comment (priytanvani) [inline frontendservice/src/views/ShopOwnerHome.vue:15]: Same here.

File: frontendservice/src/views/CustomerHome.vue (hunk 1 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/CustomerHome.vue]: Understood. Done.

File: frontendservice/src/views/ShopOwnerHome.vue (hunk 2 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/ShopOwnerHome.vue]: Done.

**Pull request merged**